runtime/onert/backend/train/ops/FullyConnectedLayer.h

```diff
@@ -59,7 +59,6 @@ class FullyConnectedLayer : public exec::train::ITrainableFunction,
   const IPortableTensor *_back_prop_output;
 
   // TODO Optimize memory
-  std::unique_ptr<Tensor> _transposed_weights;
```

Comment:
Does `Tensor` used only in the specific kernel have to be registered in TensorRegistry?

Reply:
Is it too expensive to do heap allocation every time?